examples/spa/associative_memory.ipynb

```diff
@@ -0,0 +1,499 @@
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "with spa.SPA('CleanupThreshold', seed=1) as model_3:\n",
+      "    model_3.assoc_mem = spa.AssociativeMemory(input_vocab=vocab, threshold=0.7)\n",
+      "    \n",
+      "    input_expr = '0.9*APPLE + 0.5*CHERRY + 0.4*APRICOT'\n",
+      "    model_3.am_input = spa.Input(assoc_mem=input_expr)\n",
+      "\n",
+      "    input_probe = nengo.Probe(model_3.assoc_mem.input)\n",
+      "    output_probe = nengo.Probe(model_3.assoc_mem.output, synapse=0.03)\n",
+      "\n",
+      "sim = nengo.Simulator(model_3)\n",
+      "sim.run(0.2)\n",
+      "t = sim.trange()\n",
+      "\n",
+      "input_data = sim.data[input_probe]\n",
+      "output_data = sim.data[output_probe]\n",
+      "\n",
+      "plot_similarities(input_data, output_data, vocab, top=1.1)"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "We see that the vector at the input is mostly similar to the semantic pointer `APPLE` and somewhat similar to the semantic pointer `CHERRY`.\n",
+      "The vector at the output is very similar to the `APPLE`, indicating that the memory successfully cleaned up the noisy pattern. We can indicate that we want to completely recover the pattern at the output by setting the parameter `threshold_output` to `True`. This will produce a vector at the output which has similarity one with the semantic pointer `APPLE`. "
+     ]
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "In some modelling scenarios we might have an input vector which is very similar to several other vectors in the vocabulary:"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "with spa.SPA('Cleanup', seed=1) as model_2:\n",
+      "    model_2.assoc_mem = spa.AssociativeMemory(input_vocab=vocab)\n",
+      "    \n",
+      "    # noisy input\n",
+      "    input_expr = '0.9*APPLE + 0.85*CHERRY + 0.7*APRICOT'\n",
+      "    model_2.am_input = spa.Input(assoc_mem=input_expr)\n",
+      "    \n",
+      "    input_probe = nengo.Probe(model_2.assoc_mem.input)\n",
+      "    output_probe = nengo.Probe(model_2.assoc_mem.output, synapse=0.03)"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "sim = nengo.Simulator(model_2)\n",
+      "sim.run(0.2)\n",
+      "t = sim.trange()\n",
+      "\n",
+      "input_data = sim.data[input_probe]\n",
+      "output_data = sim.data[output_probe]\n",
+      "\n",
+      "plot_similarities(input_data, output_data, vocab)"
```

Comment:
No, not really. It depends on generated vectors. I guess this is a good argument for setting a fixed axis range.